diff --git a/OrbitControls.js.bak b/OrbitControls.js
index 7408c86..aee2cc3 100644
--- a/OrbitControls.js
+++ b/OrbitControls.js
@@ -21,7 +21,7 @@
 //      controls.target.z = 150;
 // Simple substitute "OrbitControls" and the control should work as-is.
 
-THREE.OrbitControls = function ( object, domElement ) {
+OrbitControls = function ( object, domElement ) {
 
 	this.object = object;
 	this.domElement = ( domElement !== undefined ) ? domElement : document;
@@ -326,25 +326,31 @@ THREE.OrbitControls = function ( object, domElement ) {
 	}
 
 	function onMouseDown( event ) {
+        // alt or command = pan
+        // control or shift = zoom
+        var pan_key = event.altKey || event.metaKey || event.ctrlKey;
+        var zoom_key = event.shiftKey;
+        var no_key = !(pan_key || zoom_key);
+
 
 		if ( scope.enabled === false ) return;
 		event.preventDefault();
 
-		if ( event.button === 0 ) {
+		if ( event.button === 0 && no_key ) {
 			if ( scope.noRotate === true ) return;
 
 			state = STATE.ROTATE;
 
 			rotateStart.set( event.clientX, event.clientY );
 
-		} else if ( event.button === 1 ) {
+		} else if ( event.button === 1 || zoom_key ) {
 			if ( scope.noZoom === true ) return;
 
 			state = STATE.DOLLY;
 
 			dollyStart.set( event.clientX, event.clientY );
 
-		} else if ( event.button === 2 ) {
+		} else if ( event.button === 2 || pan_key ) {
 			if ( scope.noPan === true ) return;
 
 			state = STATE.PAN;
@@ -643,5 +649,3 @@ THREE.OrbitControls = function ( object, domElement ) {
 	this.update();
 
 };
-
-THREE.OrbitControls.prototype = Object.create( THREE.EventDispatcher.prototype );
